packages/puppeteer/src/getConfiguration.ts

```diff
@@ -99,31 +108,35 @@ export const getConfiguration = (): Configuration => {
   configuration.skipDownload =
     getBooleanEnvVar('PUPPETEER_SKIP_DOWNLOAD') ?? configuration.skipDownload;
 
-  // Set skipChromeDownload explicitly or from default
-  configuration.skipChromeDownload =
-    getBooleanEnvVar('PUPPETEER_SKIP_CHROME_DOWNLOAD') ??
-    configuration.skipChromeDownload;
-
-  // Set skipChromeDownload explicitly or from default
-  configuration.skipChromeHeadlessShellDownload =
-    getBooleanEnvVar('PUPPETEER_SKIP_CHROME_HEADLESS_SHELL_DOWNLOAD') ??
-    configuration.skipChromeHeadlessShellDownload;
-
   // Prepare variables used in browser downloading
   if (!configuration.skipDownload) {
-    configuration.browserRevision =
-      process.env['PUPPETEER_BROWSER_REVISION'] ??
-      configuration.browserRevision;
-
-    configuration.downloadBaseUrl =
-      process.env['PUPPETEER_DOWNLOAD_BASE_URL'] ??
-      configuration.downloadBaseUrl;
+    configuration.chrome.version =
+      process.env['PUPPETEER_CHROME_VERSION'] ?? configuration.chrome.version;
+    configuration.chrome.downloadBaseUrl =
+      process.env['PUPPETEER_CHROME_BASE_URL'] ??
+      configuration.chrome.downloadBaseUrl;
+    configuration.chrome.skipDownload =
+      getBooleanEnvVar('PUPPETEER_CHROME_SKIP_DOWNLOAD') ??
+      configuration.chrome.skipDownload;
+    configuration.chrome.skipHeadlessShellDownload =
+      getBooleanEnvVar('PUPPETEER_CHROME_SKIP_HEADLESS_SHELL_DOWNLOAD') ??
```

Comment:
This var name is strange, the product name is chrome-headless-shell. Let's keep the previous variable names in form `PUPPETEER_SKIP_$BROWSERNAME_DOWNLOAD`